lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "packages": [
+    "raiden-cli", "raiden-dapp", "raiden-ts"
+  ],
+  "version": "0.0.0"
```

Comment:
Looks like lerna can be used to bump all packages version simultaneously? Do we need to keepthis version here in sync with current version to use that on prepare_release?

Reply:
Yeah it can do that, we would need to play around and investigate how/if we want to use that

Reply:
It can be used for independent versions as well. If all versions are synced it seems to be called [fixed/locked mode](https://github.com/lerna/lerna#fixedlocked-mode-default)